payonidhi/if_else/medical_cause.cpp

```diff
@@ -0,0 +1,15 @@
+#include <iostream>
+using namespace std;
+int main() {
+  char ch;
+  cout<<"do you suffer medical illnes(y/n): "<<endl;
+  cin>>ch;
+  int a;
+  a = ch;
+  if(a==121){
```

Comment:
Similarly, this is not readable code. You can directly compare `ch == 'Y' || ch == 'y'`.